keyboards/massdrop/alt/keymaps/emptyflask/keymap.c

```diff
@@ -0,0 +1,304 @@
+#include QMK_KEYBOARD_H
+
+enum my_keycodes {
+    U_T_AUTO = SAFE_RANGE, // USB Extra Port Toggle Auto Detect / Always Active
+    U_T_AGCR,              // USB Toggle Automatic GCR control
+    DBG_TOG,               // DEBUG Toggle On / Off
+    DBG_MTRX,              // DEBUG Toggle Matrix Prints
+    DBG_KBD,               // DEBUG Toggle Keyboard Prints
+    DBG_MOU,               // DEBUG Toggle Mouse Prints
+    MD_BOOT,               // Restart into bootloader after hold timeout
+    HK_COSL,               // Clear held-down keys
+    QWERTY,                // Switch to QWERTY layout
+    COLEMAK,               // Switch to Colemak layout
+    DVORAK,                // Switch to Dvorak layout
+    WORKMAN,               // Switch to Workman layout
+};
+
+enum my_layers {
+    _QWERTY = 0,
+    _COLEMAK,
+    _DVORAK,
+    _WORKMAN,
+    _FUNCTION,
+    _NUMPAD,
+    _LAYOUTS,
+};
+
+#define TG_NKRO  MAGIC_TOGGLE_NKRO // Toggle 6KRO / NKRO mode
+#define CTL_ESC  LCTL_T(KC_ESC)    // Tap for ESC, hold for CTRL
+#define MD_LOCK  LCTL(LGUI(KC_Q))  // MacOS lock screen shortcut
+#define MO_FUNC  MO(_FUNCTION)     // Hold for function layer
+#define TG_NUMP  TG(_NUMPAD)       // Toggle numpad layer
+#define OSL_LAY  OSL(_LAYOUTS)     // One-shot layer to change layout
+
+keymap_config_t keymap_config;
+
```

Comment:
This *should* be able to be removed.
```suggestion

```

Reply:
I've verified that this still works after removing this line.